apollo-sample/src/main/java/com/example/apollostack/sample/DroidDetailsData.java

```diff
@@ -0,0 +1,26 @@
+package com.example.apollostack.sample;
+
+import com.google.auto.value.AutoValue;
+
+import com.squareup.moshi.JsonAdapter;
+import com.squareup.moshi.Moshi;
+
+import javax.annotation.Nullable;
+
+import generatedIR.DroidDetails;
+
+@AutoValue
+public abstract class DroidDetailsData implements DroidDetails.Data {
+  @Override @Nullable public abstract Species species();
+
+  public static JsonAdapter<DroidDetailsData> jsonAdapter(Moshi moshi) {
+    return new AutoValue_DroidDetailsData.MoshiJsonAdapter(moshi);
+  }
+
+  @AutoValue
+  public abstract static class Species implements Specy {
```

Comment:
Seems odd that they have it here too: https://github.com/cesarferreira/kotlin-pluralizer/blob/322fdd6d0e3e3da1bf59657a4e64bda8acc79ad5/library/src/main/kotlin/com/cesarferreira/pluralize/Pluralize.kt#L87

Oh, just glancing at this line, https://github.com/cesarferreira/kotlin-pluralizer/blob/322fdd6d0e3e3da1bf59657a4e64bda8acc79ad5/library/src/main/kotlin/com/cesarferreira/pluralize/Pluralize.kt#L48

They use `contains` which case sensitive.